python/paddle/fluid/layers/detection.py

```diff
@@ -564,6 +565,48 @@ def __reshape_to_2d(var):
     return loss
 
 
+def prior_box(input,
+              image,
+              min_sizes,
+              max_sizes,
+              aspect_ratios,
```

Comment:
Sometimes, `aspect_ratios` is not necessary, the user may not specify this argument.

```
aspect_ratios=None
```

Reply:
Done